<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="button_yes">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>440</y>
     <width>93</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HY헤드라인M</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;HY헤드라인M&quot;;</string>
   </property>
   <property name="text">
    <string>예</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_no">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>440</y>
     <width>93</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;HY헤드라인M&quot;;</string>
   </property>
   <property name="text">
    <string>아니오</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>491</width>
     <height>271</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>한컴산뜻돋움</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(241, 241, 241)</string>
   </property>
   <property name="title">
    <string>예약 여부 확인</string>
   </property>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>451</width>
      <height>221</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>한컴산뜻돋움</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>추천 자리가 전부 매진되어

현재 고르신 좌석을 예매해드린 후, 

추천 자리를 예매한 고객님이 취소할 시,

그 자리를 예매해드리겠습니다.

이에 동의 하시겠습니까?</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
